checker/check_result.go

```diff
@@ -23,6 +23,8 @@ type CheckResult struct {
 	Details     []string
 	ShouldRetry bool  `json:"-"`
 	Error       error `json:"-"`
+	Description string
```

Comment:
Let's not bloat CheckResult with Description and HelpURL. We should have a separate struct for these which other packages can access as/when needed.

Reply:
The result is where the Help documentation needs to have the necessary stuff and this is what is being used in output. 

Where else do you think this could be used? 

How do you see this as bloat?

Also, what would be your suggestion?